.github/workflows/docker.yml

```diff
@@ -0,0 +1,42 @@
+name: Build & Push
+on:
+  pull_request:
+    branches:
+      - main
+      - rc*/*
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+-*" # Push events to matching v*, i.e. v20.15.10-rc5
+env:
+  registry: docker.io
+  repository: finschia/cw-gitpod
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: login to the registry
+        uses: docker/login-action@v3
+        if: github.event_name != 'pull_request'
+        with:
```

Comment:
```suggestion
jobs:
  build:
    runs-on: ubuntu-latest
    if: github.event_name != 'pull_request'
    steps:
      - uses: actions/checkout@v4
        with:
          fetch-depth: 0
      - name: login to the registry
        uses: docker/login-action@v3
        with:
```

Reply:
https://github.com/Finschia/finschia-sdk/blob/main/.github/workflows/docker.yml
Checking if building docker image is successful is needed per PR. This is common checking for Finschia.